### PR TITLE
Add support for disabling egress traffic

### PR DESCRIPTION
**Pull request by jalgraves**
## what
* Add support for disabling egress security-group rule in default security group

## why
tl;dr My boss told me to.

We had a third party do a pen test and afterwards they suggested we remove egress traffic from our DB subnets. If I'm missing something and there is another way to accomplish this without changing the module I'd be happy and grateful to hear it.

### CODE REVIEW

**Review by Gowiem — APPROVED** [review]

File: variables.tf [hunk 1]
```diff
@@ -411,3 +411,9 @@ variable "ca_cert_identifier" {
   type        = string
   default     = null
 }
+
+variable "enabled_egress" {
```
Comment (Gowiem) [inline variables.tf:415]: We have a convention to name variables `*_enabled`. Can you update this to `egress_enabled`?

**Review by Gowiem — DISMISSED** [review]

File: variables.tf (hunk 1 above)
Comment (jalgraves) [reply variables.tf]: Yep, thought I'd done that before but I got it backwards. oops

**Review by Gowiem — APPROVED** [review]

**Comment by Gowiem** [comment]
@jalgraves 
Thanks for the contribution! Released as https://github.com/cloudposse/terraform-aws-rds-cluster/releases/tag/0.50.0.

**Pull request merged**